package.json

```diff
@@ -106,20 +154,6 @@
     "webpack-merge": "^0.14.1",
     "webpack-validator": "^2.2.9"
   },
-  "dependencies": {
```

Comment:
I think we should keep dependencies, we can clearly know which are main dependencies, and which are development tools. It doesn't affect the electron-builder.

Also, `css-modules-require-hook`, `postcss` seems unnecessary.